src/test/resources/yugabyte_user_defined_types.ddl

```diff
@@ -0,0 +1,3 @@
+CREATE TYPE my_name_type AS (first_name text, last_name varchar(40));
+
+CREATE TABLE udt_table (id INT PRIMARY KEY, name_col my_name_type);
```

Comment:
replace udt with composite_type

Reply:
Done. I have removed the complete file actually, not dependent on a ddl file now.